uptane/services/inventorydb.py

```diff
@@ -131,12 +128,12 @@ def save_vehicle_manifest(vin, signed_vehicle_manifest):
   uptane.formats.SIGNABLE_ECU_VERSION_MANIFEST_SCHEMA.check_match(
        signed_ecu_manifest)
 
-  if vin not in vehicle_manifests_dic['vehicle_manifests']:
+  if vin not in vehicle_manifests['vehicle_manifests']:
     vm_list = []
     vm_list.append(signed_vehicle_manifest)
-    vehicle_manifests_dic['vehicle_manifests'][vin] = vm_list
+    vehicle_manifests['vehicle_manifests'][vin] = vm_list
   else:
-    vehicle_manifests_dic['vehicle_manifests'][vin].append(signed_vehicle_manifest)
+    vehicle_manifests['vehicle_manifests'][vin].append(signed_vehicle_manifest)
 
   # Save all the contained ECU manifests.
   # NOTE: not tested
```

Comment:
When you say not tested here, do you mean you haven't tried it with the sample manifest I gave you? The next three lines were me scribbling help on how you might do something. Have you made sure it makes sense and tried it? You could just pass this function the dictionary I gave you and see if it gets stored correctly.